test/common/testlib.py

```diff
@@ -932,6 +932,10 @@ def check_journal_messages(self, machine=None):
             # Fedora >= 30 switched to dbus-broker
             self.allowed_messages.append("dbus-daemon didn't send us a dbus address; not installed?.*")
 
+        if self.image in ['rhel-8-1', 'rhel-8-1-distropkg']:
```

Comment:
Any reason you did it like this and not with naughty? Is problem that distropkg is affected?
Known issues are easier to follow and less likely we just forget about them. Also with naughty we would not need to backport it.

Reply:
This bug affects most/all tests, and naughties just don't scale well. They are meant for ignoring one or two failed tests, not hundreds. Updating the known issue via the github API tens of thousands of times is just not practical.